### PR TITLE
doc(readme): easier setup, full step by step example

### PR DESCRIPTION
**Pull request by charlypoly**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,213 @@
...
-Stagehand is a web automation SDK that leverages LLMs, [Playwright](https://playwright.dev/), and browser techniques to achieve a low friction, cost effective, and resilient way to automate the browser. Currently, stagehand supports Node.js and Node like runtimes.
+Stagehand is an AI Web Browser SDK that combines DOM preprocessing, Playwright, and LLMs to enable you to interact with any web page using high-level instructions.
+
+```typescript
+import { Stagehand } from "@browserbasehq/stagehand";
+
+async function example() {
+  const stagehand = new Stagehand({ env: "LOCAL" });
+  await stagehand.init();
+  await stagehand.page.goto("https://www.nytimes.com/games/wordle/index.html");
```
Comment (charlypoly) [inline README.md]: I need to rework this this, I'd love to feature a screenshot of the wordle example with the script and browser running side by side

File: README.md [hunk 2]
```diff
@@ -1,73 +1,199 @@
...
+
+_Stagehand combines DOM preprocessing with chunking, Playwright, and LLMs to complete a Wordle. Learn more in the "How it works" section._
 
 ## Getting started
 
-In order to run stagehand you'll need to create a tarball via the build step. This will be provided by the Browserbase team, or can be generated following the steps below.
+### 0. Prerequisites
 
-here's an example using npm to install a local tarball:
+Stagehand is currently compatible with Node.js, Bun, and Deno.
```
Comment (jeremypress) [inline README.md:13]: I believe it but we should test!

File: README.md [hunk 3]
```diff
@@ -1,73 +1,199 @@
...
+### 2. Configure a Browser
+
+Stagehand can be used with a local or remote Browser.
+
+**Use a local browser (_recommended for development_)**
+
+To use a local Browser, install Playwright on your local machine:
+
+```bash
+pnpm exec playwright install
```
Comment (jeremypress) [inline README.md:49]: ty!

File: README.md [hunk 4]
```diff
@@ -1,73 +1,199 @@
...
+// 4. Let's check if we can play
+const playButton = await stagehand.observe("Find the play button");
 
-async function example() {
-  const stagehand = new Stagehand({
-    env: "LOCAL",
-    verbose: true,
-    debugDom: true,
+if (!playButton) {
+  console.log("Seems like you already played wordle today!");
```
Comment (jeremypress) [inline README.md:116]: did this happen when you ran twice? A new browser always had no cookies and so you can play repeatedly

File: README.md [hunk 5]
```diff
@@ -1,73 +1,199 @@
...
+  return;
+}
+
+// The `act()` method allows you to interact with the web page
+//
+// 5. we start the game
+await stagehand.act({ action: "start the game" });
+await stagehand.act({ action: "close tutorial popup" });
+
+// 6. we loop over the 5 available guesses to complete the game
```
Comment (jeremypress) [inline README.md:126]: it's actually 6 guesses, we can fix to 0 <= 5 if that's more clear.

**Pull request merged**